parallel_algorithms/P3179/P3179.bs

```diff
@@ -3503,8 +3545,7 @@ concept <i>nothrow-bidirectional-range</i> = <i>// exposition only</i>
 template&lt;class I>
 concept <i>nothrow-random-access-iterator</i> = <i>// exposition only</i>
   <i>nothrow-bidirectional-iterator</i>&lt;I> &&
-  random_access_iterator&lt;I> &&
-  <i>nothrow-sized-sentinel-for</i>&lt;I, I>;
+  random_access_iterator&lt;I>;
```

Comment:
This is not a proper change. The `random_access_iterator<I>` concept requires `sized_sentinel_for<I,I>`, therefore *`nothrow-random-access-iterator`* should similarly require *`nothrow-sized-sentinel-for`*.

Reply:
Pardonnez moi, done!